.env-ci

```diff
@@ -1,5 +1,5 @@
-PG_DB="purspg"
```

Comment:
There was a problem with double quotes in docker-compose, so I deleted it.
This change seems to be fine for reading dotenv in purescript code.

↓The problem before change
```
$ docker-compose config
services:
  db:
    environment:
      POSTGRES_DB: '"purspg"'
      POSTGRES_PASSWORD: '"qwerty"'
      POSTGRES_USER: '"init"'
    image: postgres:12.5
    ports:
    - 5432:5432/tcp
version: '3.0'
```